public/JS/login.js

```diff
@@ -17,24 +17,24 @@ button.addEventListener('click', (e) => {
             .then((response) => {
                 console.log(response)
                 if (!response.success) {
-                    let p_tag = document.createElement('p');
-                    let p_text = document.createTextNode('Something went wrong');
-                    p_tag.appendChild(p_text);
-                    messageDisplay.appendChild(p_text);
+                    let p_tag = document.createElement('p')
+                    let p_text = document.createTextNode('Something went wrong')
+                    p_tag.appendChild(p_text)
+                    messageDisplay.appendChild(p_text)
                 } else {
-                    window.location.href = "/";
+                    window.location.href = "/"
                 }
             })
     } else {
-        let p_tag = document.createElement('p');
-        let p_text = document.createTextNode('Email not recognised');
-        p_tag.appendChild(p_text);
-        messageDisplay.appendChild(p_text);
+        let p_tag = document.createElement('p')
+        let p_text = document.createTextNode('Email not recognised')
+        p_tag.appendChild(p_text)
+        messageDisplay.appendChild(p_text)
     }
 })
 
 function isEmail(email) {
     // email regex from http://emailregex.com - "Email Address Regular Expression That 99.99% Works."
-    let regEx = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-    return regEx.test(email);
+    let regEx = /^(([^<>()\[\]\\.,:\s@"]+(\.[^<>()\[\]\\.,:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
need to add semicolons back :
`let regEx = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/`